src/Aspire.Hosting/MySql/MySqlServerResource.cs

```diff
@@ -53,9 +53,9 @@ internal void AddDatabase(string name, string databaseName)
         _databases.TryAdd(name, databaseName);
     }
 
-    internal void WriteToManifest(ManifestPublishingContext context)
+    internal async Task WriteToManifest(ManifestPublishingContext context)
     {
-        context.WriteContainer(this);
+        await context.WriteContainer(this).ConfigureAwait(false);
```

Comment:
WriteContainerAsync. Cancellation token? (or is it on the the context?)

Reply:
It's on the context